modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -308,10 +308,13 @@
     <string name="player_sleep_custom_minus">Custom minus</string>
     <string name="player_sleep_custom_plus">Custom plus</string>
     <string name="player_sleep_end_of_episode">End of episode</string>
+    <string name="player_sleep_end_of_episode_singular">End of 1 episode</string>
+    <string name="player_sleep_end_of_episode_plural">End of %d episodes</string>
     <string name="player_sleep_timer_title">Sleep Timer</string>
     <string name="player_sleep_timer">Sleep timer</string>
     <string name="player_sleep_timer_start_failed">Unable to start timer. Please allow \'Alarms &amp; reminders\' permission.</string>
-    <string name="player_sleep_when_episode_ends">Sleeping when episode ends</string>
```

Comment:
Nitpick: Just removing a string from this main strings file is sufficient. It will get automatically removed from other translations when they're updated.

Reply:
hummm... Got it! Thanks. I did not know that